project-generator/snippets/deploy-contract.js

```diff
@@ -0,0 +1,15 @@
+test("Deploy ##GENERATE-CONTRACT-NAME## contract", async () => {
+    let deployContract;
+    // ##GENERATE-TO-ADDRESS##
```

Comment:
Should this file be a handlebars template?

Reply:
Nice spot, this file should be deleted - I've reimplemeted it with Handlebars.